libs/core/src/scss/components/_grid.scss

```
@@ -0,0 +1,97 @@
+@use 'sass:map';
+
+/* stylelint-disable custom-property-pattern */
+$breakpoint-map: (
+  'xs': 0,
+  'sm': 632px,
+  'md': 768px,
+  'lg': 992px,
+  'xl': 1200px,
+);
+
+.kirby-grid {
+  --kirby-grid-columns: 12;
+  --kirby-grid-columnSpacing: 24px;
```

Consider sticking to kebab case throughout, to align better with existing custom properties. 
```suggestion
  --kirby-grid-column-spacing: 24px;
```

---

Goes for this whole file 😃